modules/outbound-command-event-handler/test/integration/application/process_sdk_outbound_bulk_accept_party_info.test.ts

```diff
@@ -0,0 +1,301 @@
+
+    // Simulate the domain handler sending the command handler PProcessPartyInfoCallback messages
+    // for each individual transfer
+    const processPartyInfoCallbackMessageData1: IProcessPartyInfoCallbackCmdEvtData = {
+      bulkId: bulkTransactionId,
+      content: {
+        transferId: randomGeneratedTransferIds[0],
+        partyResult: {
+          party: {
+              partyIdInfo: {
+                  partyIdType: 'MSISDN',
+                  partyIdentifier: '123456',
+                  fspId: 'receiverfsp'
+              }
+          },
+          currentState: 'COMPLETED'
+        },
+      },
+      timestamp: Date.now(),
+      headers: []
+    }
+    const processPartyInfoCallbackMessageData2: IProcessPartyInfoCallbackCmdEvtData = {
+      bulkId: bulkTransactionId,
+      content: {
+        transferId: randomGeneratedTransferIds[1],
+        partyResult: {
+          party: {
+              partyIdInfo: {
+                  partyIdType: 'MSISDN',
+                  partyIdentifier: '123456',
+                  fspId: 'receiverfsp'
+              }
+          },
+          currentState: 'COMPLETED'
+        },
+      },
+      timestamp: Date.now(),
+      headers: []
+    }
+
+    const processPartyInfoCallbackMessageObjOne = new ProcessPartyInfoCallbackCmdEvt(processPartyInfoCallbackMessageData1);
+    await producer.sendCommandEvent(processPartyInfoCallbackMessageObjOne);
+    const processPartyInfoCallbackMessageObjTwo = new ProcessPartyInfoCallbackCmdEvt(processPartyInfoCallbackMessageData2);
+    await producer.sendCommandEvent(processPartyInfoCallbackMessageObjTwo);
+    await new Promise(resolve => setTimeout(resolve, messageTimeout));
+
+    // Simulate the domain handler sending the command handler ProcessSDKOutboundBulkPartyInfoRequestComplete message
+    const processSDKOutboundBulkPartyInfoRequestCompleteCommandEventData : IProcessSDKOutboundBulkPartyInfoRequestCompleteCmdEvtData = {
+      bulkId: bulkTransactionId,
+      timestamp: Date.now(),
+      headers: []
+    }
+    const processSDKOutboundBulkPartyInfoRequestCompleteCommandEventObj = new ProcessSDKOutboundBulkPartyInfoRequestCompleteCmdEvt(
+      processSDKOutboundBulkPartyInfoRequestCompleteCommandEventData
+    );
+    await producer.sendCommandEvent(processSDKOutboundBulkPartyInfoRequestCompleteCommandEventObj);
+    await new Promise(resolve => setTimeout(resolve, messageTimeout));
+
+    // Check that the global state to be DISCOVERY_COMPLETED
+    const bulkState = await bulkTransactionEntityRepo.load(bulkTransactionId);
+    expect(bulkState.state).toBe('DISCOVERY_COMPLETED');
+
+    // Check domain events published to kafka
+    const hasAcceptPartyEvent = (domainEvents.find((e) => e.getName() === 'SDKOutboundBulkAutoAcceptPartyInfoRequestedDmEvt'));
+    expect(hasAcceptPartyEvent).toBeTruthy();
+
+    // Command event for bulk party info request completed
```

Comment:
Minor suggestion, can we please update the comment to say bulk accept party info

Reply:
Done. Here and throughtout.